packages/core/test/test-util/test-services/openapi/swagger-yaml-service/schema/test-entity.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved.
+ *
+ * This is a generated file powered by the SAP Cloud SDK for JavaScript.
+ */
+    
+    export type TestEntity = {
+      'stringProperty'?: string;
+      'integerProperty'?: number;
+    } | Record<string, any>;
```

Comment:
Question: I thought we agreed on not to add this allowed property.

Reply:
You probably mean the additional property? We agreed to do it for objects. additionalproperty is by default true in those cases.

Reply:
ah ok